### PR TITLE
Update docs about facets

### PR DESCRIPTION
**Pull request by bjarnef**
Updating docs about facets as `Facet()` method has changes to `FacetString()`.

Furthermore updated to new range methods using `FacetDoubleRange()`, `FacetFloatRange()` and `FacetDoubleRange()`.

### CODE REVIEW

**Comment by bjarnef** [comment]
@nzdev @nikcio I have updated this part of the documentation to match the latest changes.

File: docs/searching.md [hunk 1]
```diff
@@ -362,14 +362,14 @@ var addressFacetResults = results.GetFacet("Address"); // Returns the facets for
 
 Numeric range facets can be used with numbers and get facets for numeric ranges. For example, it would group documents of the same price range.
 
-There's two categories of numeric ranges - `DoubleRanges` and `Int64Range` for double/float values and int/long/datetime values respectively.
+There's two categories of numeric ranges - `DoubleRange` and `Int64Range` for double/float values and int/long/datetime values respectively.
```
Comment (bjarnef) [inline docs/searching.md:365]: @nikcio under the hood `double/float` may be handled equivalently, but the examples show three different facet ranges `DoubleRange`, `FloatRange` and `Int64Range`.

Comment (bjarnef) [reply docs/searching.md]: I have updated that 🙂

**Comment by Shazwazza** [comment]
Great! I Just need to cherry pick this to the docs branch.

**Pull request merged**